src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java

```diff
@@ -79,4 +79,23 @@ public CarinaDescriptionPageBase login() {
         return clickLoginBtn();
     }
 
+    @Override
+    public boolean verifyFieldsArePresent() {
+        if (nameInputField.isElementPresent()&&passwordInputField.isElementPresent()
+                && maleRadioBtn.isElementPresent()
+                &&femaleRadioBtn.isElementPresent()
+        ){
+            return true;
+        }
+        return false;
```

Comment:
bad practice
if you need verify a lot of fields use one by one(one method for one field)
Example: isNameFieldPresented, ....